mne/io/curry/tests/test_curry.py

```diff
@@ -333,20 +365,23 @@ def _mock_info_file(src, dst, sfreq, time_step):
                 out_file.write(line)
 
 
+# In the new version based on curryreader package, time_step is always prioritized, i.e.
+# sfreq in the header file will be ignored and overridden by sampling interval
```

Comment:
Can we detect this mismatch and at least keep as a warning?

Reply:
check implemented